README.md

```diff
@@ -41,10 +41,28 @@ in Internet Explorer on Windows 7 for instance (because of the wrong MIME type).
 
 JavaScript Cookie supports [npm](https://www.npmjs.com/package/js-cookie) and [Bower](http://bower.io/search/?q=js-cookie) under the name `js-cookie`.
 
+- #### Bower
+  ```
+  bower install js-cookie --save
+  ```
+
+- #### NPM
+  ```
+  npm install js-cookie --save
+  ```
+
 ### Module Loaders
 
 JavaScript Cookie can also be loaded as an AMD, CommonJS or [ES6](https://github.com/js-cookie/js-cookie/issues/233#issuecomment-233187386) module.
 
+```javascript
+import Cookie from 'js-cookie';
+```
+
+```javascript
+const Cookie = require('js-cookie');
```

Comment:
I completely agree with this point. Adding it to the README might be too much. Though it seems like too small of a segment to justify a click-away wiki article.